maps/away/away_site/uueoaesa/reclamation/ihss_reclamation.dmm

```diff
@@ -4448,7 +4448,7 @@
 	name = "Hazardous Specimen Containment";
 	id = "ihss_biglizard"
 	},
-/obj/effect/map_effect/window_spawner/full/reinforced,
+/obj/effect/map_effect/window_spawner/full/reinforced/indestructible,
```

Comment:
I am unsure about adding indestructible items to "normal" player ships.

Reply:
issue is the tyrant has p high damage and agros on sight. wasnt sure how best to handle it otherwise

Reply:
could blast doors that are closed by default help?